Frontend/ui-customer/src/Flow.purs

```diff
@@ -1113,8 +1114,9 @@ homeScreenFlow = do
                 addLocToCurrLoc state.homeScreen.props.sourceLat state.homeScreen.props.sourceLong state.homeScreen.data.source
               else pure unit
             else do
+              let isWhitelisted = ( any (_ == (getValueToLocalNativeStore MOBILE_NUMBER)) (getNumbersToWhiteList ""))
```

Comment:
can we get number from window variable - fetching from native store is expensive for each serviceability checks